service/geopm_service.proto

```diff
@@ -0,0 +1,139 @@
+//
```

Comment:
This would be cleaner if we added it as a make file target: we could identify the dependency between the output of protoc and the geopm_service.proto file.  This could enable the automatic regeneration of the output files if the proto file changed.  

Here, I'm just assuming that autogen.sh is always run first, and the user "knows" to rerun protoc-gen.sh if the geopm_service.proto file changes.  The output source files are included in the distribution tarball, so this step is not required when building from a distribution tarball (potentially without running autogen.sh).